changedetectionio/forms.py

```diff
@@ -375,6 +375,7 @@ class watchForm(commonSettingsForm):
         'Send a notification when the filter can no longer be found on the page', default=False)
 
     notification_muted = BooleanField('Notifications Muted / Off', default=False)
+    notification_screenshot = BooleanField('Include Screenshot', default=True)
```

Comment:
You have `False` in the `model/Watch.py` and `True` here, you can set this to False too

```suggestion
    notification_screenshot = BooleanField('Include Screenshot', default=False)
```